### PR TITLE
Cardboard masks

### PR DESCRIPTION
**Pull request by GoldenCan**
## About the PR
Added a "Cardboard Mask" which can be constructed using 1 sheet of cardboard and 5x LV cables.

## Why / Balance
I personally believe the cardboard mask would fit in the game, particularly with how simple it is. I have little more to say.

## Media
zCardboard Mask 1
zCardboard Mask 2
zCardboard Mask 3
(There is a typo in the first and second screenshot ("masks" instead of "mask") - it has been fixed and now reads correctly.)

## Requirements
- [x] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl:
- add: New cardboard masks can now be constructed out of a sheet of cardboard and a few low-voltage cables.

### CODE REVIEW

**Comment by dvir001** [comment]
How is this a ``BreathMask``?

**Comment by GoldenCan** [comment]
> How is this a `BreathMask`?

Perhaps you are breathing through a tube made out of an electrical cable with the wire removed? I am not entirely sure myself - and i had as such considered not making it one.

The main reason i made it a `BreathMask` was that other similiar masks were `BreathMask`-s. The primarily influence being the very similiar "joy mask", which seems to be only a flat piece of either paper, plastic or cardboard, held up by string, which can also be using as a breathing mask.
(The clown mask and mime mask being `BreathMask`-s are other influence, although less so as they are more dissimilar to a mask made out of cardboard and electrical cables.)

I am open to making changes.

File: Resources/Prototypes/Recipes/Construction/clothing.yml [hunk 1]
```diff
@@ -118,3 +118,14 @@
   description: Can hold up to 15 arrows, and fits snug around your waist.
   icon: { sprite: Clothing/Belt/quiver.rsi, state: icon }
   objectType: Item
+
+- type: construction
+  name: cardboard mask
+  id: CardboardMask
+  graph: MaskCardboard
+  startNode: start
+  targetNode: maskCardboard
```
Comment (slarticodefast) [inline Resources/Prototypes/Recipes/Construction/clothing.yml:127]:
```suggestion
  targetNode: mask
```

File: Resources/Prototypes/Recipes/Construction/Graphs/clothing/cardboard_mask.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+﻿- type: constructionGraph
+  id: MaskCardboard
+  start: start
+  graph:
+  - node: start
+    edges:
+    - to: maskCardboard
```
Comment (slarticodefast) [inline Resources/Prototypes/Recipes/Construction/Graphs/clothing/cardboard_mask.yml:7]:
```suggestion
    - to: mask
```

File: Resources/Prototypes/Recipes/Construction/Graphs/clothing/cardboard_mask.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+      steps:
+      - material: Cable
+        amount: 5
+        doAfter: 2
+      - material: Cardboard
+        amount: 1
+        doAfter: 2
+  - node: maskCardboard
```
Comment (slarticodefast) [inline Resources/Prototypes/Recipes/Construction/Graphs/clothing/cardboard_mask.yml:15]:
```suggestion
  - node: mask
```

**Review by slarticodefast — COMMENTED** [review]
Your construction graph is a little broken, causing the tests to fail. 

Since all other masks (even joy mask, which is just a piece of paper) have the BreathMask component I think it is fine to keep it for consistency.

**Comment by UbaserB** [comment]
The sprite looks a little flat, something about it feels strange but I can't put my finger on what.

**Comment by slarticodefast** [comment]
I have brought this addition up for discussion with the other maintainers and the general agreement was that this item is not needed. The main reason stated is that there are already similar, more visually interesting indentity blocking items, for example the cardborg mask. Feel free to continue to contribute in the future. Thank you.

**Pull request closed without merging**